.changeset/irc-display-nickname.md

```diff
@@ -0,0 +1,9 @@
+---
+"fuelrats.com": minor
+---
+
+Add ability to set IRC display nickname in user profile
+
+- Users can now select which of their registered IRC nicknames should be their display nickname
+- Added a star icon button next to each non-display nickname to set it as the display nickname
+- The current display nickname is indicated by the absence of action buttons
```

Comment:
Thinking on this more, I think this could be potentially confusing to a newcomer.

What if we used a filled in star next to the display name, and then the "set display" button uses an outline star? https://fontawesome.com/v5/icons/star?f=classic&s=regular

Reply:
perhaps we also give that "display" star some description on hover to explain what it means

Reply:
Could we update this to reflect the final UX?